packit/config/package_config.py

```diff
@@ -197,39 +196,13 @@ def __eq__(self, other: object):
         )
 
 
-class PackageConfigValidation:
+def get_local_package_config_content(
+    *directory, try_local_dir_first=False, try_local_dir_last=False,
+) -> Tuple[Path, Dict]:
     """
-    Package config validation
-    Does not validate fields: jobs`
-    """
-
-    def __init__(self, package_config: CommonPackageConfig):
-        self.package_config: CommonPackageConfig = package_config
-
-    def validate(self) -> dict:
-        from packit.schema import PackageConfigSchema
-
-        config_file_name_full = self.package_config.config_file_path
-        try:
-            loaded_config = safe_load(open(config_file_name_full))
-        except Exception as ex:
-            logger.error(f"Cannot load package config {config_file_name_full!r}.")
-            raise PackitConfigException(f"Cannot load package config: {ex!r}.")
-
-        try:
-            return PackageConfigSchema().validate(loaded_config)
-        except ValidationError as e:
-            return e.messages
+    Find local .packit.yaml, load it and return it
 
-
-def get_local_package_config(
-    *directory,
-    repo_name: str = None,
-    try_local_dir_first=False,
-    try_local_dir_last=False,
-) -> PackageConfig:
-    """
-    :return: local PackageConfig if present
+    :return: tuple (Path to the config, Dict with the file content)
```

Comment:
I would try to avoid using tuples as return types. (But I see that it is handy here.)

(What about having `path` attribute in the config(s). It can be sometimes helpful to see which one is used.)

Reply:
to be honest, I prefer my solution - this returns Dict, not PackageConfig, so adding "magic" key to the dict makes things more confusing from my PoV

PackageConfig.config_file_path is what you want and I don't wanna set it artificially - I just spent ~10 minutes doing that and it makes the code look pretty nasty

so unless you have an idea how to do this nicely, I'd stick to my solution

Reply:
> to be honest, I prefer my solution - this returns Dict, not PackageConfig, so adding "magic" key to the dict makes things more confusing from my PoV

No, I don't want to update the content (=dictionary).

> PackageConfig.config_file_path is what you want and I don't wanna set it artificially - I just spent ~10 minutes doing that and it makes the code look pretty nasty

What is nasty? Just introduce and fill another attribute. (We don't do any `__dict__` magic now.)

(It will not solve this specific problem but can still be useful.)

> so unless you have an idea how to do this nicely, I'd stick to my solution

What about splitting the code:
- One function for finding the correct config path.
- One function accepting path and returning content.